rust/libits-client/src/reception/exchange/mobile_perceived_object.rs

```diff
@@ -100,7 +112,7 @@ fn compute_id(object_id: u8, cpm_station_id: u32) -> u32 {
     }
 }
 
-fn compute_position(
+fn compute_position_from_mobile(
```

Comment:
why to rename?

Reply:
For consistency, as all vehicle relative computation methods are suffixed the same to make it explicit about what they do